content/docs/_index.md

```diff
@@ -1,21 +1,28 @@
 ---
-title: Overview
+title: gRPC Documentation
```

Comment:
Should this just be "Documentation"? gRPC feels a little repetative.

Reply:
Done